modules/services/cloud-connector-ecs/permissions.tf

```diff
@@ -70,7 +70,7 @@ data "aws_iam_policy_document" "iam_role_task_policy" {
       "sqs:DeleteMessageBatch",
       "sqs:ReceiveMessage"
     ]
-    resources = [module.cloud_connector_sqs.cloudtrail_sns_subscribed_sqs_arn]
+    resources = [var.cloudtrail_s3_sns_sqs_url == null ? module.cloud_connector_sqs[0].cloudtrail_sns_subscribed_sqs_arn : var.cloudtrail_s3_sns_sqs_url]
```

Comment:
this is not required, as it references the ecs role itself for the single-acount setup.
for org, the permissions must be given to the assumedRole in the management account